src/cloud/reference/cli-ref-topic.md

```diff
@@ -59,7 +59,7 @@ The following list of commonly used Magento Cloud CLI commands includes required
 Command | Description
 ------- | -----------
 `magento-cloud login` | Log in to the project.
-`magento-cloud list` | List the environments in the current project.
```

Comment:
Hi @vijay-wagento. Thank you for the fix, it makes sense. 
Could I ask you to leave both commands but with the different descriptions, please? 

```
`magento-cloud list` | List the available commands for CLI tool.
`magento-cloud environment:list` | List the environments in the current project.
```

Thank you!

Reply:
Hello, @rogyar I have added that command in the list as per suggest.

Reply:
Done